src/schema/types/resource.type.ts

```diff
@@ -138,6 +138,22 @@ export const ResourceType = new GraphQLObjectType({
         return forms;
       },
     },
+    form: {
```

Comment:
@estelafs 
you could also see it as a filter on the forms field
I'd prefer to implement this other solution